tooling/api/src/window.ts

```diff
@@ -579,6 +579,31 @@ class WindowManager extends WebviewWindowHandle {
     })
   }
 
+  /**
+   * Gets the window's current minimized state.
+   * @example
+   * ```typescript
+   * import { appWindow } from '@tauri-apps/api/window';
+   * const minimized = await appWindow.isMinimized();
+   * ```
+   *
+   * @returns Whether the window is minimized or not.
+   * */
+  async isMinimized(): Promise<boolean> {
+    return invokeTauriCommand({
```

Comment:
you also need to add the implementation for this invoke in `core/tauri/src/endpoints/window.rs`